React/wbooks/src/scss/book.scss

```diff
@@ -0,0 +1,60 @@
+.book-info {
```

Comment:
Each component should have a styles.scss file with only the classes it uses. Move all these classes to their corresponding components